garble/mpz-garble/src/evaluator/mod.rs

```diff
@@ -60,6 +61,10 @@ struct State {
     received_values: HashMap<ValueId, ValueType>,
     /// Values which have been decoded
     decoded_values: HashSet<ValueId>,
+    /// Pre-transferred garbled circuits
+    ///
+    /// (inputs, outputs) => garbled circuit
```

Comment:
Ok, then maybe
```suggestion
    /// inputs and outputs from `receive_garbled_circuit` are used for one of the circuits from `garbled_circuits`
```

Reply:
That's true. Do you have a shorter but more understandable idea? I do not think that the current comment is easily understandable, because I looked for `input` and `output` in the state variables, could not find them and was confused.